src/overlays/actors/ovl_En_Bal/z_en_bal.c

```diff
@@ -4,6 +4,24 @@
  * Description: Tingle with Balloon
  */
 
+/*
+Week Event Flags:
+        10 Bit 0 -- Whether player has talked to Tingle before during this cycle
+        35 Bits 0-5	--	Whether player has bought each possible map. For a given map, the flag is 1 << mapId.
+        64 Bits 3-4 --  Appears to be a 2 bit enum that stores player form when 10/0 is set. 0 - Zora, 1 - Deku, 2 -
+Goron, 3 - Human Notably neither bit is ever cleared by EnBal, so this field could be corrupted if not cleared at the
+same time as 10/0
```

Comment:
I think this information would be better put in `z64save.h` where the appropriate weekEventFlags are defined. I think we have everything we need to name some of these weekEventRegs, but it's not required in this PR if you don't want

Reply:
I certainly wouldn't mind naming the week events. Is it okay if I put my grubby hands on `z64save.h` (or any other files that might utilize these events) to do that?

Reply:
Yeah, go for it! If anything goes astray, it'll be caught in review anyway